internal/cmd/fluxtest-harness-influxdb/test.go

```diff
@@ -0,0 +1,244 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/cmd/flux/cmd"
```

Comment:
Depending on the `cmd` package in another project is kind of dubious - to solve the finalizer issue, could the flux test harness finalizer call stay in `flux/cmd`, but the `cmd.TestCommand` move into a library that doesn't call the finalizer.

Reply:
Though I'm not too upset if we leave the hack in the first cut

Reply:
I tried to rearrange things in Flux but hit a wall. Was able to rearrange some code in influxdb to avoid the duplicate-init panic & delete my previous hack, though